services/hexpm/hexpm.service.js

```diff
@@ -149,7 +149,9 @@ function DownloadsForInterval(interval) {
 
     async handle({ pkg }) {
       const json = await this.fetch({ pkg })
-      return this.constructor.render({ downloads: json.downloads[interval] })
+      return this.constructor.render({
+        downloads: json.downloads[interval] || 0,
```

Comment:
I've a slight preference to apply the defaults in the Joi object instead. i.e. `Joi.number().integer().default(0)` We're using that in a few places with arrays.

What do you think?